### PR TITLE
beatport.com support

### LINKED ISSUE #142: beatport.com request
We should enable StoPlay for beatport.com

### PR DESCRIPTION
**Pull request by endway**
Fixes #142

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,6 +2,10 @@
   "name": "stoplayExt-build",
   "version": "1.1.63",
   "author": "Alex Buznik",
+  "scripts": {
+    "build": "grunt rollup",
+    "bumpup": "grunt bumpup"
```
Comment (beshur) [inline package.json]: Why do we need it here?
It's only used with the release.

File: src/content/index.js [hunk 2]
```diff
@@ -214,7 +214,11 @@ class Provider {
                 break;
             case "musicforprogramming.net":
                 songName = safeGetElementTextContentByQuery('.pad a');
-                artistName = "Music for Programming"
+                artistName = "Music for Programming";
+                break;
```
Comment (beshur) [inline src/content/index.js]: OMG 🤯

File: package.json (hunk 1 above)
Comment (endway) [reply package.json]: I usually use it for development purposes. The update of the plugin from dir doesn't work good without changing the version.

Comment (beshur) [reply package.json]: but why adding it to NPM scripts?
why not `grunt bumpup` ?

Comment (endway) [reply package.json]: I don't have grunt installed globally. With this workaround, it needed only locally.

Comment (beshur) [reply package.json]: it's kind of edge-case
`./node_modules/.bin/grunt` always works

**Pull request merged**